common.txt

```diff
@@ -0,0 +1,151 @@
+aiohttp==3.7.4.post0
+amqp==5.1.1
+appdirs==1.4.4
+asgiref==3.5.2
+async-generator==1.10
+async-timeout==3.0.1
+attrs==22.1.0
+betamax==0.8.1
+betamax-serializers==0.2.1
+billiard==3.6.4.0
+black==22.10.0
+blessings==1.7
+boto3==1.17.36
+botocore==1.20.36
+Brotli==1.0.9
+build==0.9.0
+cached-property==1.5.2
+cachy==0.3.0
+celery==5.2.6
+certifi==2022.12.7
+cffi==1.15.1
+cfgv==3.3.1
+chardet==4.0.0
+charset-normalizer==2.1.1
+click==8.1.3
+click-didyoumean==0.3.0
+click-plugins==1.1.1
+click-repl==0.2.0
+colorama==0.4.6
+coreapi==2.3.3
+coreschema==0.0.4
+coverage==6.5.0
+distlib==0.3.6
+Django==4.0.8
+django-cache-memoize==0.1.8
+django-cors-headers==3.7.0
+django-debug-toolbar==3.7.0
+django-environ==0.4.5
+django-extensions==3.2.1
+django-filter==2.4.0
+django-redis==4.12.1
+djangorestframework==3.12.2
+dockerflow==2022.7.0
+docopt==0.6.2
+ecdsa==0.14.1
+exceptiongroup==1.0.4
+filelock==3.8.0
+first==2.0.2
+flake8==6.0.0
+freezegun==1.2.2
+furl==2.1.0
+gunicorn==20.0.4
+h11==0.14.0
+identify==2.5.9
+idna==3.4
+importlib-metadata==5.1.0
+iniconfig==1.1.1
+itypes==1.2.0
+Jinja2==2.11.3
+jmespath==0.10.0
+json-e==4.4.1
+jsonschema==3.2.0
+kombu==5.2.4
+loguru==0.5.3
+MarkupSafe==1.1.1
+mccabe==0.7.0
+mock==4.0.3
+mohawk==1.1.0
+mozci==1.12.6
+mozfile==2.1.0
+mozlog==7.1.0
+mozterm==1.0.0
+multidict==5.1.0
+mypy-extensions==0.4.3
+mysqlclient==2.0.3
+newrelic==5.22.1.152
+nodeenv==1.7.0
+numpy==1.23.1
+orderedmultidict==1.0.1
+outcome==1.2.0
+packaging==21.3
+pathspec==0.10.2
+pep517==0.13.0
+pip-tools==6.10.0
+platformdirs==2.5.4
+pluggy==1.0.0
+pre-commit==2.20.0
+prompt-toolkit==3.0.29
+pyasn1==0.4.8
+pycodestyle==2.10.0
+pycparser==2.21
+pycryptodome==3.10.1
+pyflakes==3.0.1
+pyparsing==3.0.9
+PyPOM==2.2.4
+pyrsistent==0.17.3
+PySocks==1.7.1
+pytest==7.2.0
+pytest-asyncio==0.14.0
+pytest-cov==4.0.0
+pytest-django==4.5.2
+pytest-freezegun @ https://github.com/hugovk/pytest-freezegun/archive/03d7107a877e8f07617f931a379f567d89060085.zip
+pytest-testmon==1.4.2
+pytest-watch==4.2.0
+python-dateutil==2.8.2
+python-jose==3.2.0
+python3-memcached==1.51
+pytz==2022.1
+PyYAML==6.0
+redis==3.5.3
+requests==2.28.2
+responses==0.22.0
+rsa==4.7.2
+s3transfer==0.3.6
+scipy==1.10.0
+selenium==4.6.0
+shellcheck-py==0.8.0.4
+simplejson==3.17.2
+six==1.16.0
+slugid==2.0.0
+sniffio==1.3.0
+sortedcontainers==2.4.0
+sqlparse==0.4.3
+taskcluster==42.1.1
+taskcluster-urls==13.0.1
+toml==0.10.2
+tomli==2.0.1
+tomlkit==0.7.0
+trio==0.22.0
+trio-websocket==0.9.2
+typed-ast==1.5.2
+types-toml==0.10.8.1
+typing_extensions==4.4.0
+tzdata==2022.6
+uritemplate==3.0.1
+urllib3==1.26.14
+vine==5.0.0
+virtualenv==20.16.7
+voluptuous==0.12.1
+watchdog==2.1.9
+wcwidth==0.2.5
+whitenoise==5.2.0
+win32-setctime==1.1.0
+wsproto==1.2.0
+yarl==1.6.3
+zipp==3.10.0
+zope.component==5.0.1
+zope.event==4.5.0
+zope.hookable==5.4
+zope.interface==5.5.2
+zstandard==0.15.2
```

Comment:
can you not pin every subdependency?  We should pin the dependencies we specifically request (like measure-noise), and rare exceptions outside of that.

Reply:
Hm, this was done automatically by the command that generates the hashes.